index.bs

```diff
@@ -431,12 +431,17 @@ The timelines present in an animator's <a>animator timeline list</a> can also af
         animator's <a>animation requested flag</a> must be set to <a>frame-requested</a> at the start
         of each animation frame.
     * If the string "scroll" is present in an animator's <a>animator timeline list</a>, then when
-        the scroll position of the nearest non-visible overflow ancestor to the animator’s
-        <a>root element</a> changes, that animator's <a>animation requested flag</a> must be set to
-        <a>frame-requested</a>.
+        the scroll position of the animator <a>root scroller</a> changes, that animator's
+        <a>animation requested flag</a> must be set to <a>frame-requested</a>.
+
+Issue: According to the current ScrollTimeline spec, the currentTime should get normalized against
+ the animation duration. We shoud match the same semantic here.
```

Comment:
BTW, this duplicates your other pull request, and will conflict with the upstream branch.